core/execution/market.go

```diff
@@ -1492,9 +1490,9 @@ func (m *Market) checkPriceAndGetTrades(ctx context.Context, order *types.Order)
 		return nil, types.OrderErrorNonPersistentOrderOutOfPriceBounds
 	}
 
-	if m.checkLiquidity(ctx, trades, persistent) {
+	/*if m.checkLiquidity(ctx, trades, persistent) {
 		return nil, types.OrderErrorInvalidPersistance
-	}
+	}*/
```

Comment:
?

Reply:
can't remember exactly why this was... had an error at some point, added this back, and commented it out later after fixing it. Will remove